Billing.API/Models/InvoiceListItem.cs

```diff
@@ -21,7 +21,7 @@ public InvoiceListItem(string product, string accountId, DateTimeOffset date, st
             Date = date;
             Currency = currency;
             Amount = amount;
-            Link = link;
+            FileName = fileName;
```

Comment:
If I am not wrong, generally, _"filename"_ is treated as a single word (see [Wikipedia](https://en.wikipedia.org/wiki/Filename))

```suggestion
            Filename = filename;
```

Reply:
good, I will change